SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -555,34 +570,47 @@ void DeleteGeneratedSchemaFiles()
 			UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("Could not clean the generated schema directory '%s'! Please make sure the directory and the files inside are writeable."), *SchemaOutputPath);
 		}
 	}
+}
+
+void CreateSchemaFolder()
```

Comment:
I think this should be `CreateGeneratedSchemaFolder` since it's not the `spatial/schema` folder right?

Reply:
Renamed.